java/core/src/main/java/co/worklytics/psoxy/gateway/impl/ApiDataRequestHandler.java

```diff
@@ -876,17 +876,16 @@ private boolean isUtf8CompatibleEncoding(String encoding) {
     @SneakyThrows
     ByteArrayContent reverseRequestBodyTokenization(@NonNull String contentType, String body) {
         // JSON case: use ObjectMapper to parse request body and map decode ON every string value
-        if (contentType.equals(ContentType.APPLICATION_JSON.toString())) {
+        if (contentType.equals(ContentType.APPLICATION_JSON.getMimeType())) {
```

Comment:
should we be more relaxed in both cases? use `contains()` instead of `equals()`?